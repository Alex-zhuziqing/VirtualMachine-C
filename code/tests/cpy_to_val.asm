FUNC LABEL 0
    MOV STK A VAL 8
    MOV VAL 1 STK A
    PRINT VAL 1
    RET
