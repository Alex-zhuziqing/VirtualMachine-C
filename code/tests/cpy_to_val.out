Cannot copy a value to the type VAL
